### PR TITLE
Fix unwanted scaling on landscape A4

### LINKED ISSUE #2: Support for A4 landscape detection
Actually on my pdftools.jar in Version 0.1.0, i got an suspect behavior if i tried to fix a4 sized landscape pdf. 

I encountered that the pdf is fixed by the application, but it should not be fixed because its A4 in landscape

I had this example file: 
[LandscapeTestFile.pdf](https://github.com/DEXPRO-Solutions-GmbH/pdftools/files/12868869/LandscapeTestFile.pdf)

### PR DESCRIPTION
**Pull request by fabiante**
This PR aims to fix #2

### CODE REVIEW

**Comment by fabiante** [comment]
Tests are not yet run, need to check why.

File: src/main/java/one/squeeze/pdftools/cli/cmds/FixPDFCommand.java [hunk 1]
```diff
@@ -71,4 +67,22 @@ private static void fix(File input, File output) throws IOException {
             pdf.save(output);
         }
     }
+
+    public static float calculateScaleFactor(PDPage page) {
+        float fWidth = MAX_WIDTH / page.getMediaBox().getWidth();
+        float fHeight = MAX_HEIGHT / page.getMediaBox().getHeight();
```
[the remark this answers is not in the document]
Comment (fabiante) [reply src/main/java/one/squeeze/pdftools/cli/cmds/FixPDFCommand.java:73]: Hm, true. I will try combining the initial `shouldPageBeRescaled` check with this calculation function.

**Comment by fabiante** [comment]
@MrBalum 

> I thought you will enable --help flag

I'll adress that in a separate PR, not this one. You can use `fix --help` which will print a help information for the `fix` command.

File: src/main/java/one/squeeze/pdftools/cli/cmds/FixPDFCommand.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (MrBalum) [reply src/main/java/one/squeeze/pdftools/cli/cmds/FixPDFCommand.java]: Thank you ! Great Work !

**Review by MrBalum — APPROVED** [review]

**Pull request merged**